run-all.sh

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Google LLC
```

Comment:
Since both this and the python script are a new way to interact with this project, this PR should also include some docs updates.

Reply:
This file could use `chmod +x`

Reply:
I think this file should also start with 

```
#!/bin/bash
```

Reply:
TODO: doc update in future PRs